.github/actions/constellation_destroy/action.yml

```diff
@@ -34,6 +34,10 @@ runs:
         # Scrap namespaces that contain PVCs
         for namespace in `kubectl get namespace --no-headers=true -o custom-columns=":metadata.name"`; do
           if [[ `kubectl get pvc -n $namespace --no-headers=true -o custom-columns=":metadata.name" | wc -l` -gt 0 ]]; then
+            if [[ "${namespace}" == "default" ]] || [[ "${namespace}" == "kube-system" ]]; then
+              kubectl delete pvc --namespace "${namespace}" --all --wait
+              continue
+            fi
```

Comment:
Do we rely on other side effects, like cleanup of LoadBalancer services? If not, we could always delete PVCs only.

Reply:
IMO, deleting the namespace is preferred for this clean up, since deleting a PVC may fail if there is still a pod using that PVC. If we just delete the entire namespace, that is no problem, if we *only* delete the pvc, we may run into some issues.

Something potentially more robust could be deleting all objects in the "default" namespace.
Cleaning up the "kube-system" namespace can be a bit more "best effort", since we shouldn't deploy any PVCs in there anyway.

I think the following should do the trick:

```bash
kubectl delete all --all --namespace default
```